baselines/exo.md

```diff
@@ -112,12 +113,14 @@ documentation provides some helpful starting points.
 #### MS.EXO.2.1v1
 A list of approved IP addresses for sending mail SHALL be maintained.
 
+<!--Policy: MS.EXO.2.1v1; Criticality: SHALL -->
 - _Rationale:_ Failing to maintain an accurate list of authorized IP addresses may result in spoofed email messages or failure to deliver legitimate messages when SPF is enabled.  Maintaining such a list ensures that unauthorized servers sending spoofed messages can be detected and permit messages from legitimate senders to be delivered.
 - _Last modified:_ June 2023
 
 #### MS.EXO.2.2v1
 An SPF policy(s) that designates only these addresses as approved senders SHALL be published.
 
+<!--Policy: MS.EXO.2.2v1; Criticality: SHALL -->
```

Comment:
Rego has 'Should/Not-Implemented'.  We need to either correct Rego, Secure Baseline document or both.  In this case I assume both should be "Shall/Not-Implemented".